src/apollo/utils/Secp256k1PrivateKey.ts

```diff
@@ -45,6 +45,7 @@ export class Secp256k1PrivateKey
     this.size = this.raw.length;
   }
 
+  // ??? duplicated in Apollo.createPrivateKey
   derive(derivationPath: DerivationPath): PrivateKey {
```

Comment:
I guess we do, the derivable abstraction requires it. I'd rather move if from the other place rather than the key abstraction.

@goncalo-frade-iohk ?

Reply:
It's something ive talked with you before this is actually not needed and not build in Swift as well.

Reply:
thanks, I'll make this the working function and get Apollo to call this, as the Apollo function does other things as well

Reply:
done